test/ecto/adapters/sqlite3/connection_test.exs

```diff
@@ -2370,9 +2370,20 @@ defmodule Ecto.Adapters.SQLite3.ConnectionTest do
   end
 
   test "create table with an unsupported type" do
+    msg =
```

Comment:
I actually think it is fine to just return the multi string error message regardless of OTP version.

Reply:
The error message afaik is not in our control. The difference is due to improvments on error messages for NIFs and BIFs, which are part of OTP24.

Reply:
Ah I see. I don't think inside the NIF we raise any exceptions there, just return atom error tuples. 👍🏻